GestureHandler.js

```diff
@@ -182,6 +194,9 @@ function createHandler(handlerName, propTypes = null, config = {}) {
     }
 
     _onGestureHandlerEvent = event => {
+      if (shouldHandleInspection) {
```

Comment:
if we set it like that would that even work if GH is setup with an Animated.event?

Reply:
Not really, but I think it's fine enough